ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -130,7 +122,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     result+=$?
     echo -e "" >> wget.log
     echo -e "[DEBUG] Downloading 'tflite label' with wget command ..." >> wget.log
-    wget -a wget.log https://raw.githubusercontent.com/nnsuite/testcases/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt
+    wget -a wget.log https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt
```

Comment:
matched with line 121
wget -a wget.log https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/mobilenet_v1_1.0_224_quant.tflite
wget -a wget.log https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt

Reply:
[**History**] FYI. We have changed the web address style because the wget command in the firewall environment could not normally re-linked from github.com/.../raw/... to https://raw.githubusercontents.com/nnsuite/..... I am not sure that temporal (=virtual address) is stable to keep the stable operation even though we move the network environment from firewall to AWS. 

* Virtual address - https://github.com/nnsuite/testcases/raw/master/ 
* Real address - https://raw.githubusercontent.com/nnsuite/testcases/master/

Reply:
Now sync to real address (https://github.com/nnsuite/testcases/raw/master ....)
If this would make an error to get tf-lite model and label files, I will change these.